instrumentation/spring/spring-boot-autoconfigure-3/build.gradle.kts

```diff
@@ -0,0 +1,22 @@
+plugins {
+  id("otel.library-instrumentation")
+}
+
+// Name the Spring Boot modules in accordance with https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.developing-auto-configuration.custom-starter
+base.archivesName.set("opentelemetry-spring-boot-3")
+group = "io.opentelemetry.instrumentation"
+
+otelJava {
+  minJavaVersionSupported.set(JavaVersion.VERSION_17)
+}
+
+dependencies {
+  val springBootVersion = "3.2.4"
+  implementation("org.springframework.boot:spring-boot-starter-web:$springBootVersion")
```

Comment:
No, because RestClientCustomizer is part of spring boot, not spring framework: https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/client/RestClientCustomizer.java

So we'd have to bump the spring boot dependency itself - but we want to stay compatible with spring boot 2.7.x